frontend/src/components/Avatar.scss

```diff
@@ -0,0 +1,15 @@
+@use '../colors.scss' as *;
+
+.avatar {
+  color: white;
+  font-size: 1.5rem;
+  font-weight: 500;
+  background-color: $ttahub-medium-blue;
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  min-width: 40px;
+  min-height: 40px;
+  border: none;
+  padding: 0;
+}
```

Comment:
most of these could be utility classes (height-5, width-5, text-white, padding-0, border-0, etc)

Reply:
changed here: 63f7cdf7